tests/commands/test_init.py

```diff
@@ -63,17 +64,35 @@ def test_all_the_files(tmp_path):
 def test_bad_name(tmp_path):
     cmd = InitCommand('group')
     with pytest.raises(CommandError):
-        cmd.run(Namespace(path=tmp_path, name='1234', author="שראלה ישראל"))
+        cmd.run(Namespace(path=tmp_path, name='1234', author="שראלה ישראל", series=None))
 
 
 def test_executables(tmp_path):
     cmd = InitCommand('group')
-    cmd.run(Namespace(path=tmp_path, name='my-charm', author="홍길동"))
+    cmd.run(Namespace(path=tmp_path, name='my-charm', author="홍길동", series=None))
     assert (tmp_path / "run_tests").stat().st_mode & S_IXALL == S_IXALL
     assert (tmp_path / "src/charm.py").stat().st_mode & S_IXALL == S_IXALL
 
 
 def test_tests(tmp_path):
     cmd = InitCommand('group')
-    cmd.run(Namespace(path=tmp_path, name='my-charm', author="だれだれ"))
+    cmd.run(Namespace(path=tmp_path, name='my-charm', author="だれだれ", series=None))
     subprocess.run(["./run_tests"], cwd=str(tmp_path), check=True)
+
+
+def test_series_defaults_to_kubernetes(tmp_path):
+    cmd = InitCommand('group')
+    cmd.run(Namespace(path=tmp_path, name='my-charm', author="fred", series=None))
+
+    with (tmp_path / "metadata.yaml").open("rt", encoding="utf8") as f:
+        metadata = yaml.safe_load(f)
+    assert metadata.get("series") == ['kubernetes']
+
+
+def test_manual_overrides_defaults(tmp_path):
+    cmd = InitCommand('group')
+    cmd.run(Namespace(path=tmp_path, name='my-charm', author="fred", series=['xenial', 'precise']))
```

Comment:
One concern for me is that this doesn't expose how a user actually interacts with the tool. Because I think this means
```
charmcraft init my-charm --author=fred --series xenial --series precise
```

Which feels a bit awkward to me. Does it work if you supply:
```
charmcraft init my-charm --author=fred --series xenial,precise
```

(I have the feeling that is treated as the 'xenial,precise' series.)

Do we care to support multiple series in "init" ? I know we want to allow for multiple series in charms, but I wonder if the syntax is such that it is too clumsy to do nicely.

Reply:
yeah, i think i like the comma-separated more than the current thing. I did the current thing because the simplest and apparently recommended is horrible to me,
```python
parser.add_argument('--series', nargs='*')
```
because the way you use that is
```
charmcraft init --series xenial precise --author fred
```

but, yes, the current one is not really better. Comma-separated it is then.